de.blobbyvolley.BlobbyVolley2.yaml

```diff
@@ -0,0 +1,65 @@
+app-id: de.blobbyvolley.BlobbyVolley2
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: blobby
+
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --persist=.blobby
+
+modules:
+  # https://github.com/flathub/org.qbittorrent.qBittorrent/blob/79e4ba75b3f71db7a58f51ade240f4bf9deb93c3/org.qbittorrent.qBittorrent.yaml#L29
```

Comment:
Please remove the unneeded comments

Reply:
I kept these comments for two reasons:

- I am not author of these modules recipes. I would like to keep attribution.
- It may be easier to see how these modules recipes were created and compare them with other flatpak apps. I am newbie and it seems easier to make it possible to compare definitions with other projects.

What do you think?

Reply:
I don't think it's really useful. All code is findable via GitHub search.